lib/questionnaires/view/src/xhtml.dart

```diff
@@ -54,8 +55,14 @@ class Xhtml extends StatelessWidget {
   }) {
     _logger.trace('enter fromRenderingString $renderingString');
 
-    final xhtml = renderingString.xhtmlText;
-    final plainText = renderingString.plainText;
+    final xhtml = renderingString.xhtmlText.translate(
+      renderingString.extensions,
+      Localizations.localeOf(context),
+    );
+    final plainText = renderingString.plainText.translate(
+      renderingString.extensions,
+      Localizations.localeOf(context),
+    );
```

Comment:
I'm not sure if this is correct, since the FHIR `extensions` are added to the `text` element of questions, and may not contain any actual xhtml markup. Won't this end up replacing the html markup with a translated string instead?

I also see that down below we're calling `_extractBase64FromImgTag(xhtml);` which sounds like it may break too, although I'm not sure.

Reply:
Thank you @fjwong, yeah you are correct, xhtml should not be translated, but please correct me if I am wrong, `plaintText` might still be translated since RenderingString can be initialized by `Text`.

Reply:
@hanny-ph I think `plainText` may or may not need translation, but I'm wondering if this would be the correct place to do so. Please refer to https://github.com/sujrd/faiadashu/pull/49#discussion_r1347205071.